_component_design/tables-controls.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Tables - Controls
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Table controls refer to elements which may be present on any table variation that allows users to interact with the data contained within a table. These controls are a collection of smaller elements such as search bars, buttons, pagination controls, action cogs, amoung others. 
```

Comment:
The intro could use some cleanup:

Table controls are elements which allow users to interact with the data contained in a table. These controls include elements such as search bars, buttons, pagination controls, and action cogs among others.

Reply:
updated.